### PR TITLE
doc: Bluetooth: Mesh: Release notes for v1.8.0

### PR DESCRIPTION
**Pull request by Balaklaka**
Release notes for Bluetooth mesh in NCS v1.8.0.

Signed-off-by: Ingar Kulbrandstad <ingar.kulbrandstad@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -208,12 +209,14 @@ Bluetooth mesh samples
 
   * :ref:`bluetooth_ble_peripheral_lbs_coex` sample, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
   * Support for :ref:`zephyr:nrf21540dk_nrf52840`.
-  * :ref:`central_and_peripheral_hrs` sample.
 
 * Updated:
 
-  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340`.
```
Comment (alxelax) [inline doc/nrf/releases/release-notes-changelog.rst:215]: Shouldn't the full sample list be added here?

**Comment by alxelax** [comment]
Do we need to add here issues that have been fixed in Zephyr (those came with upmerge)?

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -200,6 +200,7 @@ Bluetooth samples
 * :ref:`peripheral_hids_mouse` and :ref:`central_uart` samples now come with the :ref:`ble_rpc_host` child image configuration overlay.
   The overlay shows how to configure an application running a serialized Bluetooth Low Energy Host.
   These samples run out the box with the :ref:`ble_rpc` library.
+* Added :ref:`central_and_peripheral_hrs` sample.
 
 Bluetooth mesh samples
```
Comment (trond-snekvik) [inline doc/nrf/releases/release-notes-changelog.rst:205]: @greg-fer this is being added under samples, but should probably get a separate section under nRF5, right?

Comment (greg-fer) [reply doc/nrf/releases/release-notes-changelog.rst:205]: We don't have nRF5 section anymore. We try to mirror code and doc sections in the release notes changelog. So BT mesh samples should go under Samples / Bluetooth samples / BT mesh samples.

Comment (trond-snekvik) [reply doc/nrf/releases/release-notes-changelog.rst:205]: Oh. Does that mean that changes to the Bluetooth Mesh subsystem actually should go under samples, like Ingar has done here, or do we need to add some new section for that? ...or should those just not be added anymore?

Comment (greg-fer) [reply doc/nrf/releases/release-notes-changelog.rst:205]: Would it be possible to add those to Protocols / BT mesh section? That section is meant for describing bigger protocol-related changes or changes that do not fit elsewhere.

Comment (trond-snekvik) [reply doc/nrf/releases/release-notes-changelog.rst:205]: I think so. @Balaklaka ?

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -208,12 +209,14 @@ Bluetooth mesh samples
...
-  * :ref:`central_and_peripheral_hrs` sample.
 
 * Updated:
 
-  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340`.
   * Updated some samples to use DTS overlays instead of KConfig configuration files for setting up external flash memory.
+  * When publishing is enabled for a model a corresponding status message shall be published immediately after the state transition ends. This had to be done by the application before.
+  * Aligned the Silvair EnOcean Proxy Server model implementation with rev 1.2 of the Silvair EnOcean Switch Mesh Proxy Server specification.
+  * Fixed an issue where Sensor Client API can be used as non-blocking by passing NULL to the arguments that are used to fill the response.
```
Comment (PavelVPV) [inline doc/nrf/releases/release-notes-changelog.rst:219]: Why all these 3 points under `Bluetooth mesh samples` section? Should they be under `Bluetooth mesh` section like it was before?

**Review by PavelVPV — COMMENTED** [review]
Should we add the following?

- Sensor Server now always samples at a fastest cadence.
- A sensor's value is now published at a fast cadence if a measured value exceeds a delta
- The light fixture sample now publishes a light value at the end of transition.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -200,19 +200,19 @@ Bluetooth samples
...
-  * :ref:`bluetooth_ble_peripheral_lbs_coex` sample, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
+  * :ref:`bluetooth_ble_peripheral_lbs_coex`, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
   * Support for :ref:`zephyr:nrf21540dk_nrf52840`.
-  * :ref:`central_and_peripheral_hrs` sample.
 
 * Updated:
 
-  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+  * :ref:`bluetooth_mesh_light`, :ref:`bluetooth_mesh_light_lc`, :ref:`bluetooth_mesh_light_switch` and :ref:`bluetooth_mesh_sensor_server` with support for :ref:`zephyr:thingy53_nrf5340`.
   * Updated some samples to use DTS overlays instead of KConfig configuration files for setting up external flash memory.
```
Comment (mia-ko) [inline doc/nrf/releases/release-notes-changelog.rst:216]: Should these be listed as well? Your choice.
I listed at least those samples that got Thingy:53 support above.

You can also remove Updated from the bullet point.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 5]
```diff
@@ -347,7 +347,13 @@ Bluetooth libraries
   * Added support for the GATT API serialization.
   * Changed the configuration option that enables the library from the :kconfig:`CONFIG_BT_RPC` to the :kconfig:`CONFIG_BT_RPC_STACK`.
 
-Added:
+* :ref:`bt_mesh` library:
```
Comment (mia-ko) [inline doc/nrf/releases/release-notes-changelog.rst:350]: I moved those three bullets to mesh library.
To me it sounded like it's where they belong.
Open for discussion.

**Review by mia-ko — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (Balaklaka) [reply doc/nrf/releases/release-notes-changelog.rst:205]: I have no idea where things should go with this layout. Does it fit under Protocols or Libraries? I put it here, so all mesh related things was one place and kept together.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 4 above)
Comment (Balaklaka) [reply doc/nrf/releases/release-notes-changelog.rst:216]: Two of them has been updated, and two has been added. Then we should be specific in what has been done for each of them. light and light_switch has been updated with non-secure and light_lc and sensor server/client has been added.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 3 above)
Comment (Balaklaka) [reply doc/nrf/releases/release-notes-changelog.rst]: They are moved to a section under Libraries.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (Balaklaka) [reply doc/nrf/releases/release-notes-changelog.rst]: Split in added and updated.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 6]
```diff
@@ -208,12 +209,14 @@ Bluetooth mesh samples
...
   * :ref:`bluetooth_ble_peripheral_lbs_coex` sample, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
   * Support for :ref:`zephyr:nrf21540dk_nrf52840`.
-  * :ref:`central_and_peripheral_hrs` sample.
 
 * Updated:
 
-  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340` in non-secure configuration.
+  * Updated some samples with support for :ref:`zephyr:thingy53_nrf5340`.
   * Updated some samples to use DTS overlays instead of KConfig configuration files for setting up external flash memory.
+  * When publishing is enabled for a model a corresponding status message shall be published immediately after the state transition ends. This had to be done by the application before.
```
[the remark this answers is not in the document]
Comment (Balaklaka) [reply doc/nrf/releases/release-notes-changelog.rst]: This came from "https://github.com/nrfconnect/sdk-nrf/pull/5968", and that is a sample change. This is really a doc change. I will remove this as it does not change any behavior. Just explained better.

File: doc/nrf/releases/release-notes-changelog.rst (hunk 2 above)
Comment (greg-fer) [reply doc/nrf/releases/release-notes-changelog.rst:205]: You can add generic entries under protocol that link to the libraries and samples sections for BT mesh with detailed changes. With the old layout, we had sections for hardware, which in the long term would mean you would have to add the same entry three or four times under each hardware section.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 7]
```diff
@@ -200,19 +200,20 @@ Bluetooth samples
...
 Bluetooth mesh samples
 ----------------------
 
 * Added:
 
-  * :ref:`bluetooth_ble_peripheral_lbs_coex` sample, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
+  * :ref:`bluetooth_ble_peripheral_lbs_coex`, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
   * Support for :ref:`zephyr:nrf21540dk_nrf52840`.
-  * :ref:`central_and_peripheral_hrs` sample.
+  * :ref:`bluetooth_mesh_light_lc` and :ref:`bluetooth_mesh_sensor_server` with support for :ref:`zephyr:thingy53_nrf5340`.
```
Comment (trond-snekvik) [inline doc/nrf/releases/release-notes-changelog.rst:212]: These samples weren't added in this release, and should probably be grouped with the light and light switch under "updated"

File: doc/nrf/releases/release-notes-changelog.rst (hunk 5 above)
Comment (trond-snekvik) [reply doc/nrf/releases/release-notes-changelog.rst:351]: I kind of agree, but this appears to fit with the rest of the structure in the document, at least

Comment (greg-fer) [reply doc/nrf/releases/release-notes-changelog.rst:351]: I agree as well, and I would prefer to have one bit section for Protocols. This said, we have to follow the modular structure principle for the NCS, and if we merge Protocols with Samples, there will be sections for Apps, Libs, Drivers, Scripts, but no section for Samples, which will be weird. Also, some samples are not bound to one specific protocol, but to libraries, and so there must be a compromise we must settle on for this. :/

**Review by PavelVPV — APPROVED** [review]

**Review by alxelax — APPROVED** [review]

**Review by mia-ko — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Pull request merged**